docs/develop-stalker-server.md

```diff
@@ -0,0 +1,30 @@
+# Guida allo sviluppo di Stalker Server
```

Comment:
Tutto sto file andrebbe riscritto in forma impersonale, ma ci penseremo

Reply:
si l'ho pensato come una cosa between us. penso servirà mettere qualcosa di simile nel manuale dello sviluppatore e così possiam partire da questo invece che da zero